apps/src/code-studio/components/pairing/StudentSelector.jsx

```diff
@@ -3,9 +3,31 @@ import React from 'react';
 import {studentsShape} from './types';
 import i18n from '@cdo/locale';
 
+const cyan = '#0094ca';
 const styles = {
   buttonLeft: {
     marginLeft: 0
+  },
+  enabled: {
+    backgroundColor: 'white',
+    color: cyan,
+    border: '2px solid',
+    borderColor: cyan,
+    fontFamily: "'Gotham 4r', sans-serif",
+    fontSize: '13px'
+  },
+  warning: {
+    color: 'red',
+    fontFamily: "'Gotham 4r', sans-serif",
+    fontSize: '12px',
+    paddingTop: '5px'
+  },
+  disabled: {
+    opacity: 0.5
+  },
+  clicked: {
```

Comment:
NIT: (i.e. up to you if you want to change it)
I generally see "clicked" meaning more of a transitory state. i.e. "this was clicked and not it's not." Do you think "selected" would make more sense here?